File: PRAK405-2510817110006-Muhammad Fadhil Lesmana.c
#include <stdio.h>

int main() {
    int n, kelipatan;
    scanf("%d %d", &n, &kelipatan);
    int total = 0;
    for (int i = 1; i <= n; i++) {
        int jumlah = 0;
        for (int j = i; j >= 1; j--) {
            int hasil = j * kelipatan;
            jumlah += hasil;
            printf("(%d * %d)", j, kelipatan);
            if (j > 1) {
                printf(" + ");
            }
        }
        printf(" = %d\n", jumlah);
        total += jumlah;
    }
    printf("%d\n", total);
    return 0;
}

File: PRAK405-2510817110006-Muhammad Fadhil Lesmana.py
n, kelipatan = map(int, input().split())

total = 0
for i in range(1, n + 1):
    baris = []
    jumlah = 0
    for j in range(i, 0, -1):
        hasil = j * kelipatan
        jumlah += hasil
        baris.append(f"({j} * {kelipatan})")
    print(" + ".join(baris), f"= {jumlah}")
    total += jumlah
print(total)
